hack/test-go.sh

```diff
@@ -315,29 +313,12 @@ checkFDs
 
 # Convert the CSVs to arrays.
 IFS=';' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
-IFS=',' read -a etcdPrefixes <<< "${KUBE_TEST_ETCD_PREFIXES}"
 apiVersionsCount=${#apiVersions[@]}
-etcdPrefixesCount=${#etcdPrefixes[@]}
-for (( i=0, j=0; ; )); do
+for (( i=0; i<${apiVersionsCount}; i++ )); do
```

Comment:
Yeah, that'd be better. I'll fix next time I mess with this file, I don't want to have to kick off another test run.